src/containers/Console.js

```diff
@@ -52,6 +58,14 @@ function mapDispatchToProps(dispatch) {
       dispatch(editorFocusedRequestedLine());
     },
 
+    onSetPreviousHistoryIndex(index) {
+      dispatch(setPreviousHistoryIndex({index}));
+    },
+
+    onStoreCurrentConsoleInput(input) {
+      dispatch(setCurrentConsoleInput({input}));
```

Comment:
Since the argument is called `payload` it seemed appropriate to wrap these in objects, but later while debugging I noticed payload was sometimes a primitive. Either way works I think